.gitignore

```diff
@@ -1,92 +1,84 @@
-*~
+/.lock-waf*
+/.waf*
+/Tools/autotest/aircraft/Rascal/reset.xml
+/Tools/autotest/ch7_mission.txt
+/Tools/autotest/jsb_sim/fgout.xml
+/Tools/autotest/jsb_sim/rascal_test.xml
+/Tools/autotest/jsbsim_fgout_0.xml
+/Tools/autotest/jsbsim_start_0.xml
+/tmp/*
 *.bin
 *.d
+*.dfu
 *.dll
 *.elf
-*.hex
-*.dfu
 *.exe
+*.generated.h
+*.hex
 *.lst
 *.o
 *.obj
 *.px4
-*.vrx
 *.pyc
 *.tlog
 *.tlog.raw
 *.vbrain
+*.vrx
 *.zip
+*~
 .*.swo
 .*.swp
+.DS_Store
+.autotools
 .built
 .context
 .cproject
 .depend
 .directory
-.DS_Store
 .metadata/
 .project
 .settings/
+.tags
+.tags_sorted_by_file
 .vagrant
-/tmp/*
-/Tools/ArdupilotMegaPlanner/3DRRadio/bin
-/Tools/ArdupilotMegaPlanner/3DRRadio/obj
-/Tools/ArdupilotMegaPlanner/APMPlannerXplanes/APMPlannerXplanes/Debug
-/Tools/ArdupilotMegaPlanner/bin
-/Tools/ArdupilotMegaPlanner/bin/APM Planner.app
-/Tools/ArdupilotMegaPlanner/bin/Debug
-/Tools/ArdupilotMegaPlanner/bin/Release/gmapcache
-/Tools/ArdupilotMegaPlanner/bin/Release/logs/
-/Tools/ArdupilotMegaPlanner/CustomImages
-/Tools/ArdupilotMegaPlanner/Msi/upload.bat
-/Tools/ArdupilotMegaPlanner/obj
-/Tools/ArdupilotMegaPlanner/resedit/bin
-/Tools/ArdupilotMegaPlanner/resedit/obj
-/Tools/ArdupilotMegaPlanner/Updater/bin
-/Tools/ArdupilotMegaPlanner/Updater/obj
-/Tools/ArdupilotMegaPlanner/wix/bin
-/Tools/ArdupilotMegaPlanner/wix/obj
-/Tools/autotest/ch7_mission.txt
-/Tools/autotest/aircraft/Rascal/reset.xml
-/Tools/autotest/jsbsim_fgout_0.xml
-/Tools/autotest/jsbsim_start_0.xml
-/Tools/autotest/jsb_sim/fgout.xml
-/Tools/autotest/jsb_sim/rascal_test.xml
+APMrover2/test.APMrover2/
+APMrover2/way.txt
 ArduCopter/Debug/
-ArduCopter/logs/
-ArduCopter/mav.parm
-ArduCopter/terrain/*.DAT
+ArduCopter/fence.txt
+ArduCopter/ral.txt
 ArduCopter/test.ArduCopter/
 ArduCopter/test/*
-ArduPlane/logs/
-ArduPlane/mav.parm
-ArduPlane/test/*
-ArduPlane/terrain/*.DAT
+ArduCopter/way.txt
 ArduPlane/test.ArduPlane/
-APMrover2/logs/
-APMrover2/mav.parm
-APMrover2/test.APMrover2/
-mk/PX4/ROMFS/default.parm
-autotest.lck
-build
-Build.ArduCopter/*
-Build.ArduPlane/*
+ArduPlane/test/*
+ArduPlane/way.txt
 Build.APMrover2/*
 Build.AntennaTracker/*
-cmake_install.cmake
+Build.ArduCopter/*
+Build.ArduPlane/*
 CMakeCache.txt
 CMakeFiles
+LASTLOG.TXT
+Make.dep
+Thumbs.db
+autotest.lck
+build
+cmake_install.cmake
 config.mk
+cscope.in.out
+cscope.out
+cscope.po.out
 dataflash.bin
 eeprom.bin
 index.html
-LASTLOG.TXT
-Make.dep
+logs/
 mav.log
 mav.log.raw
+mav.parm
+mk/PX4/ROMFS/default.parm
 module.mk
 serialsent.raw
 status.txt
 tags
+terrain/
 test.ArduCopter/*
-Thumbs.db
```

Comment:
ah sorry, mistakes testing with code, i copy & paste from the master, i don't see this.